.env.example

```
@@ -15,11 +15,11 @@ SSM_PARAMS_PREFIX="/gurumi-bot/apps"
 SSM_CACHE_TTL_SECONDS=300
 
 # --- LLM / text (default: openai / gpt-4o-mini) ---
-LLM_PROVIDER="openai"            # openai | xai | bedrock
-LLM_MODEL="gpt-5.4"              # OpenAI: gpt-5.5  |  xAI: grok-4-3  |  Bedrock: us.anthropic.claude-opus-4-6-v1 / amazon.nova-pro-v1:0
+LLM_PROVIDER="openai"            # openai | xai | bedrock | upstage
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)